specifications/api/swagger.yaml

```diff
@@ -5065,6 +5065,17 @@ x-errNonNullRewards: &errNonNullRewards
       type: string
       enum: ['non_null_rewards']
 
+x-errMissingRewardAccount: &errMissingRewardAccount
+  <<: *responsesErr
+  title: missing_reward_account
+  properties:
+    message:
+      type: string
+      description: May occur when there is a missing reward account but wallet participates in staking.
```

Comment:
Same as in the other comment, users would run into `ErrDelegationInvalid` but never this, modulo DB malfunctions?

Reply:
yes, `MissingRewardAccount` is now err500 and can occur when there is no reward account in db (for example db malfunction), and `DelegationInvalid` is user error (err403) occurring when user instantiate shared wallet with only spending script and creates tx with delegation action.